src/styles/blocks/header.scss

```diff
@@ -0,0 +1,19 @@
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+  height: 60px;
```

Comment:
It is forbidden to set sizes and margins for blocks, but for block elements are allowed. The block should stretch to accommodate its children